MekHQ/src/mekhq/campaign/market/unitMarket/AbstractUnitMarket.java

```diff
@@ -191,8 +191,20 @@ public abstract void addOffers(final Campaign campaign, final int number, UnitMa
     public String addSingleUnit(final Campaign campaign, final UnitMarketType market,
             final int unitType, final MekSummary mekSummary,
             final int percent) {
+        final LocalDate INVASION_WAVE_FIVE = LocalDate.of(3051, 11, 1);

```

Comment:
How about we turn this into a variable players can manage. Most of the Clan equipment starts to hit later per Merc 3055, After The Battle of Tukayyid in May of 3052.  

This isn't a thing for this PR but we should also be modelling that prior to this most houses (FRR and Kurita being the exception) weren't giving salvage rights to Mercs but exchange.

Reply:
Happy to change this to after Tukayyid, but not interested in adding new campaign options until after CO IIC is merged as it just makes extra work for me to incorporate into CO IIC.

Reply:
Works for me.